SPECS/gcc/gcc.spec

```diff
@@ -149,32 +140,35 @@ export CFLAGS
 export CXXFLAGS
 export FCFLAGS
 
-SED=sed \
 %configure \
     --enable-shared \
+             LD=ld                    \
     --enable-threads=posix \
     --enable-__cxa_atexit \
     --enable-clocale=gnu \
-    --enable-languages=c,c++,fortran \
+    --enable-languages=c,c++ \
```

Comment:
no fortran?  I recall a package or two needed that downstream.  Hopefully its in extended or something...or resolved permanently.
Ok to track this elsewhere, just to get this checkedin

Reply:
I can't find anything that actually needs it. I was thinking we remove it (since it'll take a bit more work to enable) and we can always re-add upon request

Reply:
I've filed a bug to track this: https://microsoft.visualstudio.com/OS/_workitems/edit/47485967